debian/patches/0003-force-cpp11.patch

```diff
@@ -0,0 +1,11 @@
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -13,7 +13,7 @@
+ set(LIBSMU_VERSION ${LIBSMU_VERSION_MAJOR}.${LIBSMU_VERSION_MINOR}.${LIBSMU_VERSION_PATCH})
+ 
+ if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
+-	set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -pedantic")
++	set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -pedantic -std=c++11")
```

Comment:
This is already enabled by set(CMAKE_CXX_STANDARD 11); however, if we really must lower the minimum required version to accommodate Debian stable that should be removed and probably make the flag conditionally used for the compilers that recognize it (i.e. it used to be set this way until I got annoyed with the warnings thrown when compiling under VS on Windows).